assets/section-footer.css

```diff
@@ -160,46 +155,94 @@
   }
 }
 
+.footer__follow-on-shop:empty,
 .footer__list-social:empty,
 .footer-block--newsletter:empty {
   display: none;
 }
 
+.footer__follow-on-shop {
+  display: flex;
+  text-align: center;
+}
+
 .footer__list-social.list-social:only-child {
   justify-content: center;
 }
 
 .footer-block__newsletter {
   text-align: center;
+  flex-grow: 1;
 }
 
 .newsletter-form__field-wrapper {
   max-width: 36rem;
 }
 
 @media screen and (min-width: 750px) {
+  /* Pushes other components to the right of the flexbox */
   .footer-block__newsletter:not(:only-child) {
     text-align: left;
+    margin-right: auto;
   }
 
   .footer-block__newsletter:not(:only-child) .footer__newsletter {
     justify-content: flex-start;
     margin: 0;
   }
 
-  .footer-block__newsletter:not(:only-child) .newsletter-form__message--success {
+  .footer-block__newsletter:not(:only-child)
+    .newsletter-form__message--success {
     left: auto;
   }
-}
 
-.footer-block__newsletter + .footer__list-social {
-  margin-top: 3rem;
+  .footer__follow-on-shop {
+    margin-bottom: 0.4rem;
+  }
+  /* Follow on shop is the first button but it has siblings*/
+  .footer__follow-on-shop:first-child:not(:last-child) {
+    justify-content: flex-start;
+    margin-right: auto;
+    text-align: left;
+  }
+
+  /*
+    All three components are present, email, Follow on Shop, and social icons.
+    Moves the FoS button next to the social icons so they appear grouped together
+  */
+  .footer__follow-on-shop:not(:first-child):not(:last-child) {
+    justify-content: flex-end;
+    text-align: right;
```

Comment:
Same here. Since email has `flex-grow: 1;` it's preventing the `follow on shop` block to take more space than it needs and pushes it in the right spot already.

Reply:
There is some small spacing issues we have for the button. The TLDR; is that the transitioned to width is larger than the original width. If we don't have this there is actually a small gap which causes some misalignment which is really visible in mobile view.

Reply:
Sounds good 👍 thanks for the context 🙂